docusaurus.config.js

```diff
@@ -1,49 +1,60 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const lightCodeTheme = require("prism-react-renderer/themes/github");
+const darkCodeTheme = require("prism-react-renderer/themes/dracula");
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'Prefab.Cloud Documentation',
-  tagline: 'API Docs for Prefab.Cloud',
-  url: 'https://docs.prefab.cloud',
-  baseUrl: '/',
-  onBrokenLinks: 'throw',
-  onBrokenMarkdownLinks: 'warn',
-  favicon: 'img/favicon.png',
+  title: "Prefab.Cloud Documentation",
+  tagline: "API Docs for Prefab.Cloud",
+  url: "https://docs.prefab.cloud",
+  baseUrl: "/",
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "warn",
+  favicon: "img/favicon.png",
+  plugins: [
+    [
+      "posthog-docusaurus",
+      {
+        apiKey: "phc_zbGOXnt4C5OR1aKcUFyH5DgA4N0h5wOsWIsPeAuu24B",
+        appUrl: "https://app.posthog.com", // optional
+        enableInDevelopment: true, // optional
+        // other options are passed to posthog-js init as is
+      },
+    ],
+  ],
```

Comment:
This `plugins` entry is the actual change to the config.

Looks like my Prettier VSCode plugin decided to reformat the config file...not sure what we have for a linting config on this repo?